website/_data/roadmap.yml

```diff
@@ -1,37 +1,51 @@
 - status: upcoming
-  title: Smart contract and Decentralized Storage Platform
-  date: Q1 2025
+  title: Pactus Onchain Governance
+  date: Q1 2026
+
+- status: upcoming
+  title: Decentralized Domain Name System
+  date: Q4 2025
+
+- status: upcoming
+  title: Decentralized Password Manger
+  date: Q4 2025
 
 - status: upcoming
   title: Browser Extension
-  date: Q1 2025
+  date: Q3 2025
+
+- status: upcoming
+  title: Pactus Virtual Machine (PVM)
+  date: Q3 2025
+
+- status: upcoming
+  title: Mobile Friendly GUI
+  date: Q2 2025
+
+- status: upcoming
+  title: Decentralized Storage Platform
```

Comment:
What we mean by `Decentralized Storage Platform` we are not aiming to be a storage layer like filecoin or chia coin...

Reply:
It is one of the goal of Pactus to let user purchase storage on Pactus.

Reply:
I think it needs clarifications and discussion before adding to the roadmap IMHO.

Reply:
OK, Let's talk about it in the next Dev-meeting.

Reply:
Good idea, I agree.